apps/meteor/ee/server/services/authorization/service.ts

```diff
@@ -4,9 +4,6 @@ import { api } from '../../../../server/sdk/api';
 import { Authorization } from '../../../../server/services/authorization/service';
 import { getConnection } from '../mongo';
 
-getConnection()
-	.then((db) => api.registerService(new Authorization(db)))
-	.catch((error) => {
-		console.error(error);
-		process.exit(1);
-	});
+getConnection().then((db) => {
```

Comment:
no `catch` here? 👀

Reply:
I've moved the `catch` to on `getConnection` 😉

Reply:
Oh nice, i'm not a fan of calling a process.exit on a promise chain 🙈 but i guess it won't affect anything.